assets/locales/en/main.json

```diff
@@ -20,7 +20,8 @@
       "change": "Change width of a segment you’re pointing at<br />(hold <kbd class='key'>Shift</kbd> for more precision)"
     },
     "share": {
-      "sign-in": "<a href='/twitter-sign-in?redirectUri=/just-signed-in'>Sign in with Twitter</a> for nicer links to your streets and your personal street gallery",
+      "sign-in-twitter": "<a href='#'>Sign in with Twitter</a>",
+      "sign-in": " for nicer links to your streets and your personal street gallery",
```

Comment:
Also, since we're changing an existing translation, it might be better to create a new translation key? I know one thing that can get weird when having html in translations is that different language have different orders for sentence structure. It might not make sense to have the link fixed at the beginning of the sentence.

Reply:
I think we can pass in JSX as a value https://github.com/yahoo/react-intl/blob/master/examples/translations/src/client/components/greeting.js So I would think we could use a value for the link itself that way it could still be moved around when it's translated

Reply:
Yup -- we can replace placeholders with JSX. So the string might look like this:

```
    "sign-in": "{signInLink} for nicer links to your streets and your personal street gallery",
```

and different languages can decide where to move the placeholder.

For "sign-in-twitter" we might as well add a generic one that's just "Sign in" because we'll very soon evolve past just having Twitter as a login method.